.github/workflows/advice.js

```diff
@@ -41,7 +41,9 @@ ${codespacesBadge}
 #### Install mlflow from this PR
 
 \`\`\`
-pip install git+https://github.com/mlflow/mlflow.git@refs/pull/${issue_number}/merge
+# Use \`%sh\` to run this command on Databricks
+OPTIONS=$(if pip freeze | grep -q 'mlflow @ git+https://github.com/mlflow/mlflow.git'; then echo '--force-reinstall --no-deps'; fi)
```

Comment:
`--force-reinstall` reinstall everything and slow down the installation. To minimize the impact, use `--no-deps` assuming all the mlflow requirements are already installed.

Reply:
https://docs.databricks.com/en/libraries/notebooks-python-libraries.html#can-i-use-sh-pip-pip-or-pip-what-is-the-difference

> On Databricks Runtime 11.3 LTS and above, %pip, **_%sh pip_**, and !pip all install a library as a notebook-scoped Python library.